pkg/cmd/server/server.go

```diff
@@ -144,6 +147,12 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringSliceVar(&config.restoreResourcePriorities, "restore-resource-priorities", config.restoreResourcePriorities, "desired order of resource restores; any resource not in the list will be restored alphabetically after the prioritized resources")
 	command.Flags().StringVar(&config.defaultBackupLocation, "default-backup-storage-location", config.defaultBackupLocation, "name of the default backup storage location")
 
+	volumeSnapshotLocations := flag.NewMap().WithKeyValueDelimiter(":")
+	command.Flags().Var(&volumeSnapshotLocations, "default-volume-snapshot-locations", "list of unique volume providers and default volume snapshot location (provider1:location-01, provider2:location-02, ...)")
```

Comment:
let's remove the spaces in the example just after the commas (so `provider1:location-01, provider2:location-02,...`)